### ISSUE TITLE
Test Pilot icon could look better in Firefox dark theme

### ISSUE THREAD

**Issue by alex-mayorga** [mozilla/testpilot#2832]
Steps:
- Enable Firefox's dark theme

Result:
![imagen](https://user-images.githubusercontent.com/649262/29889186-f0fea2aa-8d88-11e7-9d07-c9078d287623.png)

Expected result:
Test Pilot icon looks better.

**Comment by ChenMorpheus** [comment]
Thanks for the suggestion. Peko, the visual designer, will work on this to provide asset here.

**Comment by davehunt** [comment]
I'm not sure it deserves a separate issue, but the icon doesn't look too great in the light theme either:

<img width="177" alt="screenshot 2017-09-12 16 50 54" src="https://user-images.githubusercontent.com/122800/30347723-00636f4e-97db-11e7-8b76-5d082b0c7390.png">

**Comment by ghost** [comment]
related to #2858

**Comment by Pekochen** [comment]
![1505902270](https://user-images.githubusercontent.com/8252151/30638932-a9d18e94-9e2f-11e7-9d21-28cb7acb7028.png)
FYI
Thanks!!

**Comment by fzzzy** [comment]
This is now fixed in addon 2.0.4, which has been deployed to production.

**Comment by alex-mayorga** [comment]
¡Hola @fzzzy!

It looks much better but still not exactly the right colors...
![imagen](https://user-images.githubusercontent.com/649262/30718304-247006a2-9ee5-11e7-81df-ca90cb801596.png)

![imagen](https://user-images.githubusercontent.com/649262/30718378-6758ffa0-9ee5-11e7-8fae-d1d0013789eb.png)

Maybe I need to tune up the colors on this screen?

¡Gracias!
Alex

**Comment by alex-mayorga** [comment]
¡Hola!

The Library icon is #BEBEBE with #A3A3A4 for the edges in the dark theme, and the Test Pilot icon is #F9F9FA instead.

The Library icon is #5A5B5C with #A3A3A4 for the edges in the light theme, and the Test Pilot icon is #18191A instead.

¡Gracias!
Alex

**Comment by SoftVision-PaulOiegas** [comment]
I can confirm what @alex-mayorga mentioned. We have seen it to yesterday, but due to the amount of testing needed yesterday (both stage and production servers) we forgot to inform you guys about this.

I'm not sure if this was intended or not but the Test Pilot button now looks very prominent from other buttons.  I will reopen this until we decide if there is anything to be done in this case.

**Comment by ChenMorpheus** [comment]
Hi all, 

I think this guideline below is more fit with the current design, and it specifically defines what colors for icons in dark/light theme and the states. You can find [here](http://design.firefox.com/photon/visual/icons.html#color) for more details. 
![2017-09-22 6 02 26](https://user-images.githubusercontent.com/18019946/30739521-480299aa-9fc0-11e7-8fe4-deb648d91d7f.png)

**Comment by SoftVision-PaulOiegas** [comment]
This seems to be correctly fixed now. 👍 It has the same colors codes as @alex-mayorga mentioned in the comment and like the other icons around it, that are displayed by default in Firefox.

However, those color codes are not the ones that @ChenMorpheus mentioned in the above comment. but since the other ones have the same color, I assume that he might be wrong on this.
 
Verified this on latest Nightly build (2017-09-25) using all 3 themes and it seems the icon correctly adapts like other Firefox toolbar buttons on each theme. So marking this as fixed.

**Comment by ChenMorpheus** [comment]
@SoftVision-PaulOiegas it's great to see the bug is fixed now. However, I just want to confirm with you about the guideline I provided since it could be a great opportunity to highlight to Design System. 

> However, those color codes are not the ones that @ChenMorpheus mentioned in the above comment. but since the other ones have the same color, I assume that he might be wrong on this.

If I understand correctly, what you meant is the guideline from Design System is not the correct color codes compared to other icons, right?

**Comment by SoftVision-PaulOiegas** [comment]
@ChenMorpheus Maybe I'm wrong, and please correct me if so, but it says:
- For Light background, icon color should be #0c0c0d, while my color picker says #5A5B5C
- For Dark background, icon color should be #f9f9fa, while my color picker says #BEBEBE 

Instead I see #f9f9fa for the default theme toolbar background, but not for the icon itself.

**Comment by ChenMorpheus** [comment]
@SoftVision-PaulOiegas Thanks for your response. I've highlighted the issue to Design System team, and we found that current implementation of other icons on toolbar is incorrect. That means they might update the implementation and guidelines as well. I'll let you know if there is any updates, thanks.

**Comment by SoftVision-PaulOiegas** [comment]
Wow, that's something. Probably that's why all the add-ons icons look awkward first time when implemented and only after we log issues they are fixed. 😄
Thank you too for the updates!
